readme.md

```diff
@@ -31,6 +31,8 @@ and add the private key from Firebase Admin SDK.
   (if the value of `firebase.configurations.path` is not changed, the json file bundled in the jar will be referred)
 - go into the folder where the jar is (eg. `cd target`)
 - run `java -jar covid19-1.0.0-SNAPSHOT.jar`. Check if the jar version matches.
+- the access logs can be found in folder access located in the folder where tomcat is.
+ (linux: /tmp/<latest_tomcat_folder>/access) These logs can be disabled from the application.yml

```

Comment:
we needs these access logs in the application's log folder. please check if there's a property to define location..

Reply:
We can give an absolute path of a folder withing double quotes. I just sent a new commit.